script/seed_bucket_data.py

```diff
@@ -0,0 +1,18 @@
+from calitp.storage import get_fs
```

Comment:
Please write some comments at the top of this file that explain the purpose of this file and how to use it. It might be a good idea to add a blurb in the README about this too.

Reply:
done!